recipes/pydy-all/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.4.0" %}
+
+package:
+  name: pydy-all
```

Comment:
Is there a recommended naming convention for metapackages that install optional dependencies? There is currently `pydy` which installs the required dependencies. Should this be `pydy-all`, `pydy-optional`, `pydy-full`, etc? Wondering if there is any precedent.

Reply:
It is really up to the package.  Most packages use just name `pydy` for the metapackage and have their dependency named something like `pydy-core`.  But it doesn't really matter

Reply:
Hmm, I'd have to change what the package pydy does to move to that convention, but would like to follow precedent. Thanks for the tip.

Reply:
Yeah, I wouldn't change the existing packages, just because that is even more confusing to your users.  Another thing to consider is that some packages, like NLTK, split out their data into packages like `nltk-data` that have to be installed separately.  I don't know that you have that issue.

Reply:
Cool. thanks.